skyportal/tests/fink/produce_fake.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Copyright 2021 AstroLab Software
+# Author: Julien Peloton
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
If this file is from some other repo, our general approach is to import it and use it accordingly. If you're contributing original code here you'll need to accept the licensing of the Skyportal project itself and remove this preamble. Authorship is attributed via git commits.